src/app/globals.css

```diff
@@ -0,0 +1,8 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+body {
+  background-color: #1a2a33
```

Comment:
is this correct? We probably want to use maybe one of Tailwind colors, or if Tailwind doesn't have the color that we need, we will need to add it to the tailwind.config

Reply:
I think this is `dark-green` in tailwind

you can do i believe:

```
body {
  @apply bg-dark-green
}
```